logback/logback/build.gradle

```diff
@@ -1,4 +1,4 @@
 dependencies {
     // Logback
-    api libs.logback12
+    api project(':logback12')
```

Comment:
```suggestion
    api project(':logback12')
```
Could you also check if the POM file is correctly created?

Reply:
I've double-checked it with `generatePomFileForJarPublication` command and the POM file was correctly generated.
<img width="675" alt="image" src="https://github.com/line/armeria/assets/1866157/8f2676af-55eb-4290-9b80-a617d528bcd2">

Reply:
Great, thanks!